functional_tests.go

```diff
@@ -2557,49 +3000,58 @@ func testGetObjectClosedTwiceV2() {
 
 	// Save the data
 	objectName := randString(60, rand.NewSource(time.Now().UnixNano()), "")
+	args["objectName"] = objectName
+
 	n, err := c.PutObject(bucketName, objectName, reader, "binary/octet-stream")
 	if err != nil {
-		logger().Fatal("Error:", err, bucketName, objectName)
+		failureLog(function, args, startTime, "", "PutObject failed", err).Fatal()
 	}
 
 	if n != int64(thirtyThreeKiB) {
-		logger().Fatalf("Error: number of bytes does not match, want %v, got %v\n", thirtyThreeKiB, n)
+		failureLog(function, args, startTime, "", "Number of bytes does not match, expected "+string(thirtyThreeKiB)+" got "+string(n), err).Fatal()
 	}
 
 	// Read the data back
 	r, err := c.GetObject(bucketName, objectName)
 	if err != nil {
-		logger().Fatal("Error:", err, bucketName, objectName)
+		failureLog(function, args, startTime, "", "GetObject failed", err).Fatal()
 	}
 
 	st, err := r.Stat()
 	if err != nil {
-		logger().Fatal("Error:", err, bucketName, objectName)
+		failureLog(function, args, startTime, "", "Stat failed", err).Fatal()
 	}
+
 	if st.Size != int64(thirtyThreeKiB) {
-		logger().Fatalf("Error: number of bytes in stat does not match, want %v, got %v\n",
-			thirtyThreeKiB, st.Size)
+		failureLog(function, args, startTime, "", "Number of bytes does not match, expected "+string(thirtyThreeKiB)+" got "+string(st.Size), err).Fatal()
 	}
 	if err := r.Close(); err != nil {
-		logger().Fatal("Error:", err)
+		failureLog(function, args, startTime, "", "Stat failed", err).Fatal()
 	}
 	if err := r.Close(); err == nil {
-		logger().Fatal("Error: object is already closed, should return error")
+		failureLog(function, args, startTime, "", "Object is already closed, should return error", err).Fatal()
```

Comment:
No point of logging nill error message, err:
failureLog(function, args, startTime, "", "Object is already closed, should return error"~~, err~~).Fatal()